kytos/core/controller.py

```diff
@@ -25,7 +25,8 @@
 from kytos.core.api_server import APIServer
 from kytos.core.buffers import KytosBuffers
 from kytos.core.events import KytosEvent
-from kytos.core.helpers import now, start_logger
+from kytos.core.helpers import now
+from kytos.core.log_manager import LogManager
```

Comment:
@diraol, I think that put everything in core is a bad idea. Controller ok. But Log lets split into log module.